src/index.js

```diff
@@ -79,3 +79,19 @@ Object.defineProperty(mapboxgl, 'accessToken', {
  * mapboxgl.supported() // = true
  * @see [Check for browser support](https://www.mapbox.com/mapbox-gl-js/example/check-for-support/)
  */
+
+/**
+ * Sets the map's URL Transform callback function
+ * Call this method to set a callback to transform URLs before they are requested
+ * from the internet. The callback can be used to add or remove custom parameters, or reroute
+ * certain requests to other servers or endpoints.
+ *
+ * @function setURLTransform
+ * @param {Function} callback Callback function called with the original URL to be transformed. Return the transformed URL
+ * @example
+ * mapboxgl.setURLTransform( (url)=> {
+ *  if(url.startsWith('https://myHost') {
+ *   return url += `&_ts=${Date.now()}`;
```

Comment:
I know this is just an example, but can we change it so that we also check for presence of `?` before blindly appending `&`?